rest/api_test.go

```diff
@@ -1607,18 +1680,12 @@ func TestChannelAccessChanges(t *testing.T) {
 	assertStatus(t, rt.Send(request("PUT", "/db/d1", `{"channel":"delta"}`)), 201) // seq=7
 	assertStatus(t, rt.Send(request("PUT", "/db/g1", `{"channel":"gamma"}`)), 201) // seq=8
 
-	// Artificial delay to let updates show up on changes feed
-	time.Sleep(time.Second)
 
-	// Check the _changes feed:
-	var changes struct {
-		Results []db.ChangeEntry
-	}
-	response = rt.Send(requestByUser("GET", "/db/_changes", "", "zegpold"))
-	log.Printf("_changes looks like: %s", response.Body.Bytes())
-	err = json.Unmarshal(response.Body.Bytes(), &changes)
+	numChangesExpected := 1
+	changes, err := rt.WaitForChanges(numChangesExpected, "/db/_changes", "zegpold")
+
```

Comment:
Consider changing this to a new getSequenceForDoc(docid) method, followed by waitForSequence() to avoid multiple changes requests causing false positives.